CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Fixes bare variables in conditionals (deprecation warnings with Ansible 2.8+) (#204)
 * Fix boolean values when checking array (#207)
+* Update and enhance the test environment (#215)
```

Comment:
I believe it would be good to be more expressive what this change brings (nature of changes in the tests, extended OS coverage, etc)

Reply:
Since there is a reference to the issue/pull-request (#215, commit comments are detailed), I thought that it would be enough (and handy) to keep it short (ideally fitting in a single line). I also had a look at the previous entries in CHANGELOG.md to make my proposal.

What about cherry-picking from these bits 😅:

> - Update the test environment (#205)
>     - Add following target systems: CentOS 8, Debian 10, Ubuntu 18.04 and 20.04
>     - Install Ruby 2.6.6 and 2.7.1 with RVM
> - Modify some test assertions to better abstract RVM behaviour from its internal implementation ("blackbox" testing) (#205)
> - Fix and enhance Travis CI setup (#205)
>      - Install more recent versions of Ruby, to be compatible with latest Bundler version
>      - Validate system-wide setup (root.yml) 
>      - Parallelize the tests to speed up the builds

To avoid a to long ping-pong, could you please give me the text you would put instead? You better know how you want this should look like... 😊

Reply:
Looks good! Updated the CHANGELOG